modules/microshift-data-cleaning-full-cleanup.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-full-cleanup_{context}"]
+= Cleaning all data and configuration
+
+Cleanup all the {microshift-short} data and configuration by running the `microshift-cleanup-data` script.
```

Comment:
Same comments about starting this section with "you can" and also about splitting "cleanup" into two words:
```suggestion
You can clean up all the {microshift-short} data and configuration by running the `microshift-cleanup-data` script.
```